### PR TITLE
Add PolicyAndCookieBanner

### PR DESCRIPTION
**Pull request by dworthen**

### CODE REVIEW

File: packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		script.src = COOKIE_SCRIPT_SRC
+		document.head.appendChild(script)
+
+		let totalLoadingTime = 0
+		function isLoaded(): void {
+			if (isCookieScriptLoaded()) {
+				res()
+			} else {
+				if (totalLoadingTime >= MAX_SCRIPT_LOADING_TIME_MS) {
+					throw new Error(`Error loading ${COOKIE_SCRIPT_SRC}`)
```
Comment (darthtrevino) [inline packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts:96]: What will be the effect of the banner not being shown? Should we prevent the site from loading?

File: packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+			const cookieBannerPlaceholderElement = document.getElementById(
+				COOKIE_CONSENT_BANNER_ID,
+			)!
+
+			window.WcpConsent!.init(
+				language,
+				cookieBannerPlaceholderElement,
+				function initializeConsentManager(err, consentManager) {
+					if (err) {
+						console.error(err)
```
Comment (darthtrevino) [inline packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts:59]: Should we introduce an onError() argument into the hook?

File: packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+function isCookieScriptLoaded(): boolean {
+	return (
+		!!document.getElementById(COOKIE_SCRIPT_ID) &&
+		window.WcpConsent !== undefined
+	)
+}
+
+async function addCookieScript(): Promise<void> {
+	return new Promise((res, rej) => {
```
Comment (darthtrevino) [inline packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts:83]: let's use `resolve` and `reject`

**Review by darthtrevino — APPROVED** [review]

File: packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts (hunk 1 above)
Comment (dworthen) [reply packages/themed-components/src/PolicyAndCookieBanner/PolicyAndCookieBanner.hooks.ts]: Added an onError prop for handling errors.

**Pull request merged**